### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by didaquis** [quicksilver/Quicksilver#2999]
### Before submitting your bug report, please confirm you have completed the following steps

- [X] Searched the issue is not already listed in the [Issue List](https://github.com/quicksilver/Quicksilver/issues?q=is%3Aissue) (I've searched it)
- [X] Searched in the [Quicksilver Users' Group](https://groups.google.com/forum/#!forum/blacktree-quicksilver) for questions relating to tech support.
- [X] Downloaded the [latest version of Quicksilver](https://qsapp.com/) for your system and confirmed the issue still persists.

### Bug description

Hi. 

Sometimes (not always) the clipboard history provided by "Clipboard plugin" is not displayed on the active desktop.

<img width="615" alt="Captura de pantalla 2024-01-25 a las 11 00 00" src="https://github.com/quicksilver/Quicksilver/assets/6883041/2d3116d2-45be-4ebc-b7cb-7cbf62ee2899">

### Steps to reproduce

1. using two monitors, use the mission control function to create more spaces.
2. move to the second space and invoke Quicksilver
3. copy some value (example: text from the browser)
4. Press the ⌘L shortcut to open the clipboard history
5. Observe that you can't see the clipboard history. Reason => the clipboard history is opened on the first space, but you (an quicksilver interface) are on the second space

### Expected behavior

I expect see the clipboard history on the same space than Quicksilver interface. 

### MacOS Version

macOS 13

### Quicksilver Version

2.4.2 (4041)

### Relevant Plugins

Clipbaord plugin. Version 1.8.1

**Comment by didaquis** [comment]
Any update related to this issue?

**Comment by n8henrie** [comment]
No, sorry. I don't have a second monitor to test, and I'm (slowly) trying to focus on the crashes and race conditions first.

**Comment by frimicc** [comment]
I see the same behavior with multiple desktops. Since that uses the same underlying code as multiple monitors, fixing it there might fix it for multiple monitors too.

If I'm on Desktop3 the clipboard history pane will appear in Desktop2. It's always 1 virtual Desktop "to the left" of the active desktop, for some reason.
